packages/pn-pa-webapp/src/pages/components/NewNotification/PaymentMethods.tsx

```diff
@@ -249,13 +210,36 @@ const PaymentMethods = ({
     }
   };
 
+  const formIsEmpty = (values: any) => {
+    // eslint-disable-next-line functional/no-let
+    let isEmpty = true;
+    notification.recipients.forEach(recipient => {
+      const currentDocument = values[recipient.taxId];
+      if (currentDocument.pagoPaForm && currentDocument.pagoPaForm.file.name !== '') {
+        isEmpty = false;
+      }
+      if (currentDocument.f24flatRate && currentDocument.f24flatRate.file.name !== '') {
+        isEmpty = false;
+      }
+      if (currentDocument.f24standard && currentDocument.f24standard.file.name !== '') {
+        isEmpty = false;
+      }
+    });
+    return isEmpty;
+  };
+
   const formik = useFormik({
     initialValues,
-    validationSchema: notification.paymentMode !== PaymentModel.NOTHING ? validationSchema : null,
     validateOnMount: true,
     onSubmit: async (values) => {
-      if (isCompleted) {
-        onConfirm();
+      const emptyForm = formIsEmpty(values);
```

Comment:
I was quite confused while trying to figure out the effect of the modifications. Maybe is clearer if the first conditional statement is left as it was, and we just change the condition of the second one to 
```
if (notificationPaymentMode !== PaymentModel.NOTHING && !emptyForm)
```
or perhaps even
```
if (!emptyForm)
```
adding a comment to notice that `(notificationPaymentMode === PaymentModel.NOTHING` implies `emptyForm`.

What do you think about this?